src/javaharness/java/arcs/android/demo/service/NotificationRenderer.java

```diff
@@ -70,6 +72,15 @@ public boolean render(PortableJson content) {
           PendingIntent.getService(
               context, 0, notificationIntent, PendingIntent.FLAG_UPDATE_CURRENT);
       builder.setContentIntent(pendingIntent);
+    } else if (data.hasKey(DEEPLINK_FIELD)) {
```

Comment:
can there be both - handler and deeplink? maybe log a warning?

also we probably should start documenting what rendering content field the "notification" modality supports.